eth_keys/datatypes.py

```diff
@@ -57,28 +58,47 @@
     ByteString = collections.abc.ByteString
 
 
-class BackendProxied(object):
+class LazyBackend(object):
+    def __init__(self, backend=None):
+        # type: (BaseECCBackend) -> None
```

Comment:
The code below suggests to me the `backend` arg can also be a string or a class, but here the type annotation claims it can only be an instance of `BaseECCBackend`